org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/preferences/XMLCatalogs.java

```diff
@@ -62,7 +63,18 @@ private static File getWTPExtensionCatalog() {
 			.forEach(element -> {
 				String namespace = element.getAttribute("systemId");
 				String uri = element.getAttribute("uri");
-				if (!URI.create(uri).isAbsolute()) {
+				if (uri.startsWith(PLATFORM_PLUGIN_PATH)) {
```

Comment:
I have tried to use FileLocator#resolve with URL platform but it returns a jar URI instead of file URI. I must find more time to investigate how to fix that with generic mean.

Reply:
using `FileLocator.toFileURL` as in current patch should work here; it's just that the condition to use it seems narrower than necessary, and the part that does the parsing seems redundant with what is already done in `FileLocator.toFileURL`.